bcipy/tasks/query_selection.py

```diff
@@ -0,0 +1,190 @@
+""" Query selection module. Each query method is a class to keep track of the
+ history. """
+
+import numpy as np
+import random
+
+eps = np.power(.1, 5)  # a small number to avoid numerical issues
+
+
+class RandomQuery(object):
+    """ Given the final posterior selects the N most likely options to be
+        queried.
+        Attr:
+            len_alp(int): Length of the alphabet. Query selection returns
+                integers as queries, each corresponding to the location of the
+                query to be selected
+            len_query(int): number of queries in the scheduled query
+        Functions:
+            reset():
+                this selection method is memoryless, therefore this is void\
+            update_query():
+                updates the querying method and picks the particular query
+        """
+
+    def __init__(self, len_alp, len_query=4):
+        self.alp = list(range(len_alp))
+        self.len_query = len_query
+
+    def reset(self):
+        """ This object does not require history. Reset is a void function. """
+        tmp = None
```

Comment:
For functions that don't do anything, just use the keyword `pass` (see Task.py).